packages/truffle-core/lib/repl.js

```diff
@@ -113,9 +113,25 @@ ReplManager.prototype.stop = function(callback) {
   }
 };
 
-ReplManager.prototype.interpret = function(cmd, context, filename, callback) {
-  var currentContext = this.contexts[this.contexts.length - 1];
-  currentContext.interpreter(cmd, context, filename, callback);
+ReplManager.prototype.interpret = function(
+  replInput,
+  context,
+  filename,
+  callback
+) {
+  const processedReplInput = processReplInput(replInput);
```

Comment:
Note that this approach means you can type `truffle n` to step next in the debugger, since it reuses the same ReplManager.

Not sure if this matters, but if it does, we could move this logic to `lib/console` instead. I'll think on it a bit.

Reply:
Ah ok, I didn't realize they both used this. And yes, maybe you are right that it belongs in `lib/console` if that is unused by the debugger.